{
  "comments": [
    {
      "key": {
        "uuid": "3a7867ff_ba1d19dc",
        "filename": "libappfuse/FuseBuffer.cc",
        "patchSetId": 2
      },
      "lineNbr": 54,
      "author": {
        "id": 1003224
      },
      "writtenOn": "2016-12-14T06:24:25Z",
      "side": 1,
      "message": "is this right? don\u0027t you need an \"errno \u003d -result\" or \"\u003c\u003c strerror(-result)\"?",
      "revId": "a6373ec1d47c0b370c87b3915feeba8f2b4523f7",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "fae08f14_2c027373",
        "filename": "libappfuse/FuseBuffer.cc",
        "patchSetId": 2
      },
      "lineNbr": 54,
      "author": {
        "id": 1091002
      },
      "writtenOn": "2016-12-14T06:41:02Z",
      "side": 1,
      "message": "The method needs to be invoked just after invoking read and write. For failure case, result should contain -1 and errno should contains the error number.",
      "parentUuid": "3a7867ff_ba1d19dc",
      "revId": "a6373ec1d47c0b370c87b3915feeba8f2b4523f7",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}